models/iris_depth_camera_3/iris_depth_camera_3.sdf

```diff
@@ -0,0 +1,237 @@
+<sdf version='1.5'>
+  <model name='iris_depth_camera_3'>
```

Comment:
In terms of naming: can we try to differentiate between `iris_depth_camera` and `iris_depth_camera_3` without using numbering? Like a specific thing that one has and the other doesn't? Having a `iris_depth_camera` and `iris_depth_camera_3` listed doesn't seem adequate.

Reply:
The differentiation come from the number of cameras, hence the 3.

Reply:
Yes. Check: https://github.com/PX4/sitl_gazebo/pull/358#discussion_r335365125